### DISCUSSION TITLE
Changed domain but is still accessible via old domain.

### DISCUSSION THREAD

**Discussion by apoorv569** [nextcloud/all-in-one#3865] [Questions] [answered]
I bought a new domain so I wanted change my Nextcloud instance to use the newer one, so I followed the guide here https://github.com/nextcloud/all-in-one#how-to-change-the-domain.

It worked and the new domain is working. I use `NginxProxyManager` to proxy the local domain to external domain and I still have the old domain added. I noticed that I can still access the older domain and works perfectly fine, can login, view files etc.

Is this simply because of the `NginxProxyManager` doing its thing or did I misconfigure something and the `Nextcloud` is accessible with any domain I proxy it to? If so why is that we had to change it in the configuration file?

Just curious.

**Comment by szaimen** [comment, marked as answer]
Hi, did you already restarted the containers via the aio interface after the domain change?

**Reply by apoorv569** [reply]
Yes, I did.  OK I noticed that if I go old domain, I get a 302 which according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302 is a re-direct status code with a given location header. So I think its working correctly? Is this the expected way? Or should I get `errors` or something like 404?

**Reply by szaimen** [reply]
> So I think its working correctly?

Yes it is
